doc/index.md

```diff
@@ -251,6 +251,8 @@ provides, and a blurb about how each is intended to be used.
   <DD>Type-erased range of elements with value type `T`; can store _any_ range with this value type.</DD>
 <DT>\link ranges::views::c_str_fn `views::c_str`\endlink</DT>
   <DD>View a `\0`-terminated C string (e.g. from a `const char*`) as a range.</DD>
+<DT>\link ranges::views::cache1_fn `views::cache1`\endlink</DT>
+  <DD>Caches the most recent element within the view so that dereferencing the view's iterator multiple types doesn't incur any recomputation. This can be useful in adaptor pipelines that includes combinations of `view::filter` and `view::transform`, for instance. `views::cache1` is always single-pass.</DD>
```

Comment:
```suggestion
  <DD>Caches the most recent element within the view so that dereferencing the view's iterator multiple types doesn't incur any recomputation. This can be useful in adaptor pipelines that include combinations of `view::filter` and `view::transform`, for instance. `views::cache1` is always single-pass.</DD>
```